src/server.ts

```diff
@@ -25,6 +26,9 @@ export class InversifyExpressServer  {
     ) {
         this._container = container;
         this._router = customRouter || express.Router();
+        this._routingConfig = {
+            rootPath: "/"
```

Comment:
I think it is a good idea to add an optional constructor argument for routingConfig. I know they can use `setRoutingConfig` as well but I feel like is nicer to be able to configure everything via the constructor. What do you think? Thanks!